src/platform/monitoring/DowntimeNotification/containers/DowntimeNotification.jsx

```diff
@@ -83,10 +83,12 @@ class DowntimeNotification extends React.Component {
   };
 
   render() {
-    const { appTypeContent } =
-      this.props.customText?.appType || APP_TYPE_DEFAULT;
+    const { formConfig } = this.props;
+
+    const appType = formConfig?.customText?.appType ?? APP_TYPE_DEFAULT;
+
```

Comment:
This isn't a form, so I think this may have been a copy-paste mistake.

Reply:
This `appType` is used in [this function](https://github.com/department-of-veterans-affairs/vets-website/blob/bbf5fc94c61df6cb919deafae840d82c14618eed/src/platform/monitoring/DowntimeNotification/containers/DowntimeNotification.jsx#L62) and is implemented [here](https://github.com/department-of-veterans-affairs/vets-website/blob/bbf5fc94c61df6cb919deafae840d82c14618eed/src/platform/monitoring/DowntimeNotification/containers/DowntimeNotification.jsx#L77). GitLens says you wrote [this function](https://github.com/department-of-veterans-affairs/vets-website/blob/bbf5fc94c61df6cb919deafae840d82c14618eed/src/platform/monitoring/DowntimeNotification/containers/DowntimeNotification.jsx#L63). If you're okay with it, I was thinking of using ``` const appType = formConfig?.customText?.appType ?? APP_TYPE_DEFAULT``` instead.

Reply:
The `appType` part  makes sense, but the `formConfig` part doesn't.

Reply:
I think you may be getting this confused with the other `DowntimeNotification` component in "src/platform/forms/save-in-progress/DowntimeMessage.jsx". From what I can tell on master, that one has the destructuring bug too.

Reply:
So `appType` is being used here, in the `renderGlobalDowntimeOverride` function :

```
      <AlertBox
        className="vads-u-margin-bottom--4"
        headline={`This ${appType} is down for maintenance`}
        isVisible
        status="warning"
      >
        <p>
          We’re making some updates to this {appType}. We’re sorry it’s not
          working right now and we hope to be finished by {endTime}. Please
          check back soon.
        </p>
      </AlertBox>

```
The `appType` lives in `customText` which lives in `formConfig`. Is that what you're asking about?